src/ansys/simai/core/data/models.py

```diff
@@ -108,3 +109,22 @@ def build(
                 dismiss_data_with_volume_overflow,
             )
         )
+
+    def cancel_build(self, model: Identifiable[Model]):
```

Comment:
Why is this in ModelDirectory if the model is not used (the project is) ?
Imagine this:
```
old_model = ...
new_model_in_training = ...

client.models.cancel_training(old_model)  # oops I just cancelled the new one
```

Reply:
Agreed, moved to `projects`. ONLY reason for `models` was Camille's description of [sc-25165], where he points to models docs for the missing method.